cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,8 +175,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
-        }
+        repository.delete(releaseName_build)
```

According to the fix instructions, we were given two options: either remove the catch block entirely if the exceptions have no impact, or handle the exception with a logging statement. In this case, since neither logging nor specific handling logic was provided for these exceptions, and the fix instruction included the option to remove the block, removing the empty catch block entirely is the preferred and instructed solution.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)